extensions/amp-base-carousel/1.0/amp-base-carousel.css

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-base-carousel {
+  display: block;
+  overflow: hidden;
+}
+
+amp-base-carousel > * {
+  box-sizing: border-box !important;
+  margin: 0 !important;
+  flex-shrink: 0 !important;
```

Comment:
This assumes that parent should be flex. But it's not in the above declaration?

Reply:
This is an interesting one. For the slotted elements the [Preact parent will be flex.](https://github.com/ampproject/amphtml/blob/master/extensions/amp-base-carousel/1.0/base-carousel.css.js#L22)

Reply:
This deserves a comment, IMHO.